src/main/java/com/stripe/net/Webhook.java

```diff
@@ -69,7 +94,7 @@ public static final class Signature {
      * @throws SignatureVerificationException if the verification fails.
      */
     public static boolean verifyHeader(
```

Comment:
Adding a parameter here constitutes a public API breaking change. Do you mind adding an overload?

Reply:
done, kindly help to check